studio/src/lib/schema-helpers.ts

```diff
@@ -632,6 +631,11 @@ export const extractDirectives = (node: ASTNode | undefined | null): ExtractedDi
       case "deprecated":
         const deprecatedDirValues = getArgumentValues(GraphQLDeprecatedDirective, directive);
         result.deprecationReason = (deprecatedDirValues.reason ?? "") as string;
```

Comment:
What would the type of this be if you didn't `as` it? This looks like it should be a string anyways? If not, is there a good reason why it isn't?

Reply:
Ok, but you're already doing (x ?? "whatever") so shouldn't typescript pick that up as a type assertion?

Reply:
It doesn't, it assumes the union `{} | string` sadly